Marlin/src/module/stepper.h

```diff
@@ -348,6 +351,16 @@ class Stepper {
       static constexpr bool adaptive_step_smoothing_enabled = true;
     #endif
 
+    #if ENABLED(SMOOTH_LIN_ADV)
+      static float  extruder_advance_TAU;       // The smoothing time, which is also the lookahead 
```

Comment:
I would rather see `extruder_advance_TAU` declared as a private variable along with `extruder_advance_TAU_TICKS` and `extruder_advance_ALPHA` and then read from outside the `Stepper` class through an accessor function.

Reply:
Good call. Although it was safe to read, it wasn't safe to write directly